.github/tests/namespace-override/post-install.sh

```diff
@@ -30,7 +31,7 @@ cat <<EOF >>"$GITHUB_STEP_SUMMARY"
 | spire-spiffe-oidc-discovery-provider | <pre>$(k_wait spire-server deployments.apps "${RELEASE}-spiffe-oidc-discovery-provider")</pre> |
 EOF
 
-if [ $1 -ne 0 ]; then
+if [ "$1" -ne '0' ]; then
```

Comment:
I don't understand this one. != is used for strings, -ne for integers.

Reply:
woops good catch